src/lib/containers/Error.tsx

```diff
@@ -21,12 +25,20 @@ export const Error = (props: ErrorProps) => {
         variant={'danger'}
         transition={'div'}
       >
-        {error.status === 403 && (
-          <p>
-            Please <SignInButton /> to view this resource.
-          </p>
-        )}
-        <p>{error.reason}</p>
+        <p>
+          {loginError && (
+            <>
+              Please <SignInButton /> to view this resource.
+            </>
+          )}
+          {accessDenied && (
+            <>You are not authorized to access the page requested.</>
```

Comment:
Is it always going to be a "page"?

Reply:
It would not.. maybe instead - "You are not authorized to access this resource"?

Reply:
updated the text